hooks/check

```diff
@@ -11,19 +11,32 @@ fi
 
 ok=yes
 vault="secret/$GENESIS_VAULT_PREFIX"
-domain="$(lookup params.static_ip)"
```

Comment:
instead of removing this value, you should replace it with:
```
static_ip="$(lookup params.static_IP)"
domain="$(lookup params.external_domain "$static_ip")"
```
then reference those values below.  This is because `lookup` is a very time-consuming task and will significantly slow down this script.